test/bestrefs/test_bestrefs.py

```diff
@@ -56,18 +117,18 @@ def test_normalize_id(line, expected):
     assert test_brs.normalize_id(line) == expected
 
 
-# Currently hung up on how to test the property decorated functions "only_ids" and "drop_ids"
-#@pytest.mark.parametrize('line, expected',
-#                          [
-#                              ([], [])
-#                              (['id1', 'Id2:iD2'], ['ID1:ID1', 'ID2:ID2']),
-#                          ])
-#def test_only_ids():
-#    """Test should demonstrate normalization of ID list."""
-#    test_brs = br.BestrefsScript()
-#    print(test_brs.args)
-#    print(test_brs.only_ids)
-#    assert test_brs.only_ids == expected
+def test_only_ids():
+    """Test should demonstrate only_ids is set to None."""
+    test_brs = br.BestrefsScript()
+    assert test_brs.only_ids == None
```

Comment:
are you meaning to use an equality check here?
```suggestion
    assert test_brs.only_ids is None
```

Reply:
Yes thank you.